src/mappings/curve.ts

```diff
@@ -54,3 +55,16 @@ export function handleCurveSUSDTransfer(event: RemoveLiquidity): void {
   let pricePerFullShare = vaultContract.get_virtual_price();
   createPriceHistory(vaultAddress, vaultName, totalSupply, pricePerFullShare, timestamp, txnHash)
 }
+
+
+export function handleCurveRenTransfer(event: RemoveLiquidity): void {
+  let vaultAddress  = event.address;
+  let vaultContract = CurveRen.bind(vaultAddress);
+  let vaultName     = "curve_susd";
```

Comment:
this should be curve_ren

Reply:
I wonder if there is a way to dynamically pull this in from subgraph.yaml. Something for us to think about for a future improvement.